R/mod_quality_overview.R

```diff
@@ -227,6 +228,23 @@ mod_quality_overview_server <- function(input,
         sort(decreasing = T)
     )
   })
+
+  output$appInfo <- shiny::renderUI({
```

Comment:
Use snake_case instead since the rest of the outputs are using snake_case? `app_info`

Reply:
trying my best to adapt...